packages/core/src/lib/patternlab.js

```diff
@@ -36,8 +36,7 @@ module.exports = class PatternLab {
   constructor(config) {
     // Either use the config we were passed, or load one up from the config file ourselves
     this.config =
-      config ||
-      fs.readJSONSync(path.resolve(__dirname, '../../patternlab-config.json'));
+      config || require(path.resolve(__dirname, '../../patternlab-config.js'));
```

Comment:
Couldn't we just do this instead and then either `.json` or `.js` (or even `patternlab-config/index.js`) would work? Backwards compatible so previous installs do not error if they have `.json` and might even help with some of the other problems you've mentioned.

```diff
- require(path.resolve(__dirname, '../../patternlab-config.js'))
+ require(path.resolve(__dirname, '../../patternlab-config'))
```